src/nav/full-stack-observability.yml

```diff
@@ -2583,13 +2583,15 @@ pages:
           - title: Enable Lambda monitoring
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring
             pages:
-              - title: 1. Account linking
+              - title: Intro to enabling Lambda monitoring
+                path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring
+              - title: Step 1. Account linking
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking
-              - title: 2. Instrumentation examples
+              - title: Step 2. Instrumentation examples
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example
-              - title: 3. Instrument your functions
+              - title: Step 3. Instrument your functions
```

Comment:
@zuluecho9 , how about using a colon instead of a period like this: "Step 3: Instrument your functions"?